test/ngAnimate/animationSpec.js

```diff
@@ -0,0 +1,834 @@
+'use strict';
+
+ddescribe('$$animation', function() {
```

Comment:
I remember we had a Grunt task for this which was failing a build in those cases...